azuredevops/internal/service/taskagent/resource_kubernetes_resource_test.go

```diff
@@ -0,0 +1,254 @@
+//go:build (all || resource_environment_resource_kubernetes) && !exclude_resource_environment_resource_kubernetes
```

Comment:
rename `resource_kubernetes_resource_test.go` to `resource_environment_resource_kubernetes_test.go`